hatch.toml

```diff
@@ -59,3 +59,6 @@ check-sdist = [
     "pip freeze | grep dbt-redshift",
 ]
 docker-prod = "docker build -f docker/Dockerfile -t dbt-redshift ."
+
+[tool.hatch.metadata]
```

Comment:
This can't be in `hatch.toml` nor `pyproject.toml` to publish to PyPI. `hatch build` will fail since this is not reproducible.